containers/Connector/config.ts

```diff
@@ -43,11 +49,9 @@ export const initOnboard = (network: Network, subscriptions: Subscriptions) => {
 					apiKey: process.env.NEXT_PUBLIC_PORTIS_APP_ID,
 				},
 				{ walletName: 'trust', rpcUrl: infuraRpc },
-				{ walletName: 'dapper' },
```

Comment:
And put dapper back

Reply:
BlockNative Onboard removed support for Dapper, Unilogin, and Squarelink back in v1.18.0.